mdagent/tools/base_tools/analysis_tools/inertia.py

```diff
@@ -0,0 +1,159 @@
+import os
+from typing import Optional
+
+import matplotlib.pyplot as plt
+import mdtraj as md
+import numpy as np
+from langchain.tools import BaseTool
+from pydantic import BaseModel, Field
+
+from mdagent.utils import FileType, PathRegistry
+
+
+def load_traj(path_registry, top_fileid, traj_fileid=None):
+    all_fileids = path_registry.list_path_names()
+    if top_fileid not in all_fileids:
+        raise ValueError("Topology File ID not found in path registry")
+    top_path = path_registry.get_mapped_path(top_fileid)
+
+    if traj_fileid is None:
+        return md.load(top_path)
+
+    if traj_fileid not in all_fileids:
+        raise ValueError("Trajectory File ID not found in path registry")
+
+    traj_path = path_registry.get_mapped_path(traj_fileid)
+    return md.load(traj_path, top=top_path)
+
+
+def save_to_csv(path_registry, data, file_id, description=None):
+    file_path = f"{path_registry.ckpt_records}/{file_id}.csv"
+    i = 0
+    while os.path.exists(file_path):
+        i += 1
+        file_path = f"{path_registry.ckpt_records}/{file_id}_{i}.csv"
+    np.savetxt(file_path, data, delimiter=",")
+    path_registry.map_path(file_id, file_path, description=description)
+    return file_path
```

Comment:
We'll probably need this in several places, can you generalize them & move them to utils?